apps/nextjs-website/react-components/utils/Components.helpers.tsx

```diff
@@ -0,0 +1,281 @@
+import React, { ReactElement } from 'react';
+import { Button, Typography, TypographyProps } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { CtaButtonProps } from './Components.types';
+
+{
+  /* COMMON ELEMENTS */
+}
+
+export const useBackgroundColor = (theme: 'dark' | 'light') => {
+  const { palette } = useTheme();
+  return theme === 'dark' ? palette.primary.dark : palette.background.paper;
+};
+
+export const useBackgroundColorAlternative = (theme: 'dark' | 'light') => {
+  const { palette } = useTheme();
+  return theme === 'dark' ? palette.primary.dark : palette.background.default;
+};
+
+export const useTextColor = (theme: 'dark' | 'light') => {
+  const { palette } = useTheme();
+  return theme === 'dark' ? palette.primary.contrastText : palette.text.primary;
+};
+
+export const useTextAlternativeColor = (theme: 'dark' | 'light') => {
+  const { palette } = useTheme();
+  return theme === 'dark' ? palette.primary.contrastText : palette.primary.dark;
+};
+
+export const useExtraTextColor = (theme: 'dark' | 'light') => {
+  const { palette } = useTheme();
+  return theme === 'dark'
+    ? palette.primary.contrastText
+    : palette.text.secondary;
+};
```

Comment:
Update: It could NOT actually, as theme-aware properties only work inside of the sx prop and we're not going to use the sx prop for all of the above values.